website/docs/reference/query-operators/logical-operators.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
@noisersup Please could you take a look at this setup? I don't understand why CTS is failing since the expected response is just as described here.